### PR TITLE
Add content: How to make a gift card

### PR DESCRIPTION
**Pull request by onionltd**

### CODE REVIEW

File: content/learn/make.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+slug: how-to-make
+cover:
+    image: ""
+    hidden: true
+---
+Monero gift cards are an easy way to give Monero. Made out of paper and in size of a credit card, the cards are small enough to fit in your wallet. Each gift card comes with a unique QR code. The QR code contains everything the other person needs to restore a Monero wallet and access the deposited funds. Because the information encoded in the QR code is sensitive, it's important you never reveal it to anyone else but the person that is going to receive the card.
+
+## Printing gift cards
+
+Our site previews some card templates made by the community. You can download them for free and print them at home. For best results, we highly recommend making use of a professional service. Find a printing shop in you area that prints business cards and pay them to print the cards for you. Pay extra for rounded corners, if this option is available. Be advised that printing shops will likely print only in bulk. If this is your case, you may end up with a lot of spare gift cards.
```
Comment (plowsof) [inline content/learn/make.md]: typo - "in you area" -> "in your area"

File: content/learn/make.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## Printing QR codes
+
+Card templates come without a QR code. You must print the QR codes separately and stick them onto a card. Unlike the card itself, we recommend printing QR codes only on a device which you trust. Don't underestimate importance of this advice. The information in the QR code is all that is needed to steal Monero from the card.
```
Comment (plowsof) [inline content/learn/make.md]: "don't underestimate importance" -> "don't underestimate the importance"

File: content/learn/make.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+To generate QR codes, we have built a web application that generates a page full of QR codes prepared for printing. The application works offline, nothing gets stored on our servers.
+
+As for printing, print on self-adhesive paper. If you use a laser printer, print on a paper with glossy finish to achieve the best result. Take scalpel and trace around each QR code to cut them into individual pieces. Peel off the protective layer and stick the QR code to the card.
```
Comment (plowsof) [inline content/learn/make.md]: "take scalpel" -> "take a scalpel"

File: content/learn/make.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+## Depositing Monero to a gift card
+
+Once you have a card with a QR code on it, it's time to give value to the card. Open a personal wallet that has some Monero in it. Proceed to create a new transaction, open a camera to scan a destination address from the QR code, set amount and finish by submitting the transaction to the network.
+
+The amount you give is up to you to decide. Use the current market price as a clue. It's a good practice to deposit Monero to a card shortly before gifting it. Following this practice frees you from having to remember the deposited amounts.
```
Comment (plowsof) [inline content/learn/make.md]: "its a good" "it's good"

**Comment by onionltd** [comment]
Thanks @plowsof. I've pushed updated version of the document.

**Pull request merged**